README.md

```diff
@@ -35,6 +35,15 @@ To preview the website locally:
 
    Then, load <http://localhost:4000> in your browser.
 
+## Local Setup using Docker
+
+0. Ensure that you have Docker and Make installed.
+1. Run `make build` to build the project.
+2. Run `make serve` to serve the project, this include live reloading so any change will be reflected (it can take a while, check the logs).
+3. Run `make clean` to remove the docker images and resources generated.
+
+Note: You can run `make help` to obtain detailed information on how to use our make commands.
```

Comment:
Pardon me for budding in but I wanted bring the [Alerts markdown syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) to your attention in case you weren't aware of it.

I think this `note:` is a could benefit from the `[!TIP]` callout UI.

**Example screenshots**:

_Expand the `<details>` below for demos_

<details><summary>Light theme example screenshot</summary>


<img width="1054" alt="image" src="https://github.com/expressjs/expressjs.com/assets/17415134/ab4cbad6-0a12-403b-8b67-e3a34adbb8e5">


</details> 

<details><summary>Dark theme example screenshot</summary>

<img width="1078" alt="image" src="https://github.com/expressjs/expressjs.com/assets/17415134/6b74539a-36e5-406a-9619-b6b4c7044a75">

</details> 



**Changes to produce the examples above**:

```suggestion
## Local Setup using Docker

>[!TIP]
> You can run `make help` to obtain detailed information on how to use our make commands.

0. Ensure that you have Docker and Make installed.
1. Run `make build` to build the project.
2. Run `make serve` to serve the project, this include live reloading so any change will be reflected (it can take a while, check the logs).
3. Run `make clean` to remove the docker images and resources generated.
```